### PR TITLE
iris0

### PR DESCRIPTION
**Pull request by miwanoff**
irise0 element was added

### CODE REVIEW

**Comment by arhibober** [comment]
Look nice!

**Comment by miwanoff** [comment]
Thanks!

File: index.html [hunk 1]
```diff
@@ -81,7 +89,7 @@
     <img
       class="iris"
       id="iris4"
-      style="position: absolute; left: 230px; top: 320px"
```
Comment (arhibober) [inline index.html]: To much irises!

**Review by arhibober — COMMENTED** [review]
Some more comments

**Comment by miwanoff** [comment]
I want to merge

**Pull request merged**